Breeding-Insight/bi-api
# Feature/bi 1189 - 4.0 Exp Preview & Commit: Create Exp Independent Variables

# Description
**Story:** [BI1189](https://breedinginsight.atlassian.net/browse/BI-1189)

4.0 Exp Preview & Commit: Create Exp Independent Variables


# Dependencies
bi-web branch feature/BI-1146

# Testing

- Using the Experiment Import Template, create an import file with  
-- One experiment (every row has the same _Exp Title_ and _Exp Description_)
-- several Obsevation Units
- import and commit 
- verify that the correct data is saved

# Checklist:

- [X] I have run TAF:  [this TAF run](https://github.com/Breeding-Insight/taf/actions/runs/2415896451) has the same errors as [future/1.0 run](https://github.com/Breeding-Insight/taf/actions/runs/2423124641)
